src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -17,9 +22,14 @@
     "PMD.ExcessiveMethodLength", "PMD.ClassNamingConventions", "PMD.PrematureDeclaration"})
 public class Et3ResponseHelper {
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final String TEMPLATE_NAME = "ET3_0922 (1).docx";
```

Comment:
Template naming should be in a specific format - we can get into a call and discuss this